veps/sig-compute/node-pressure-eviction.md

```diff
@@ -0,0 +1,229 @@
+# Overview
+
+Kubernetes has two types of pod eviction.
```

Comment:
Would you be able to use https://github.com/kubevirt/enhancements/blob/155c2ad57b91ed55e2634b68575fc4b4122be126/veps/NNNN-vep-template/vep.md for the VEP?